PWGJE/EMCALJetTasks/AliAnalysisTaskRhoSparse.cxx

```diff
@@ -131,57 +134,88 @@ Bool_t AliAnalysisTaskRhoSparse::Run()
   static Double_t rhovec[999];
```

Comment:
Same comment as in other pull request - the large static array should be replaced by std::vector as it is excessive resource usage and particularly generates problems in parallelized environments or when multiple objects of the same type are used in parallel.